resources/views/mship/management/dashboard.blade.php

```diff
@@ -1,7 +1,7 @@
 @extends('layout')
 
 @section('content')
-    <div class="row">
+    <div class="row no-edge-gutters">
```

Comment:
Removing col-md-12 from the child elements instead should resolve the padding issue here